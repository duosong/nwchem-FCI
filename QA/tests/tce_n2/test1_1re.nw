start tce_n2_1re
 
echo
 
geometry units bohr
  symmetry c1
  n 0 0   -1.034
  n 0 0    1.034
end
 
basis
 * library cc-pvdz
end

 
scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end
 
tce
ccsd
end
 
task tce energy
 
