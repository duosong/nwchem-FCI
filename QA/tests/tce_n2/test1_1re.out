nohup: ignoring input
 argument  1 = test1_1re.nw
  NWChem w/ OpenMP: maximum threads =    1



============================== echo of input deck ==============================
start tce_n2_1re

echo

geometry units bohr
  symmetry c1
  n 0 0   -1.034
  n 0 0    1.034
end

basis
 * library cc-pvdz
end


scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end

tce
ccsd
end

task tce energy

================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.0.1
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2020
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = koothan
    program         = /hpc/home/kowalski/nwchem/bin/LINUX64/nwchem
    date            = Wed Apr  7 16:05:04 2021

    compiled        = Sun_Jan_17_18:37:55_2021
    source          = /hpc/home/kowalski/nwchem
    nwchem branch   = 7.0.0
    nwchem revision = nwchem_on_git-2069-ga5c152f2d
    ga revision     = 5.8.0
    use scalapack   = T
    input           = test1_1re.nw
    prefix          = tce_n2_1re.
    data base       = ./tce_n2_1re.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107200 doubles =    100.0 Mbytes
    stack    =   13107197 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 n                    7.0000     0.00000000     0.00000000    -1.03400000
    2 n                    7.0000     0.00000000     0.00000000     1.03400000

      Atomic Mass 
      ----------- 

      n                 14.003070


 Effective nuclear repulsion energy (a.u.)      23.6943907157

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.09434


            XYZ format geometry
            -------------------
     2
 geometry
 n                     0.00000000     0.00000000    -0.54716928
 n                     0.00000000     0.00000000     0.54716928

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 n                |   1 n                |     2.06800  |     2.06800
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          1
 ==============================================================================





 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvdz                   on all atoms 


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  n (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  9.04600000E+03  0.000700
  1 S  1.35700000E+03  0.005389
  1 S  3.09300000E+02  0.027406
  1 S  8.77300000E+01  0.103207
  1 S  2.85600000E+01  0.278723
  1 S  1.02100000E+01  0.448540
  1 S  3.83800000E+00  0.278238
  1 S  7.46600000E-01  0.015440

  2 S  9.04600000E+03 -0.000153
  2 S  1.35700000E+03 -0.001208
  2 S  3.09300000E+02 -0.005992
  2 S  8.77300000E+01 -0.024544
  2 S  2.85600000E+01 -0.067459
  2 S  1.02100000E+01 -0.158078
  2 S  3.83800000E+00 -0.121831
  2 S  7.46600000E-01  0.549003

  3 S  2.24800000E-01  1.000000

  4 P  1.35500000E+01  0.039919
  4 P  2.91700000E+00  0.217169
  4 P  7.97300000E-01  0.510319

  5 P  2.18500000E-01  1.000000

  6 D  8.17000000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 n                          cc-pvdz                  6       15   3s2p1d


                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    30
  atoms           =     2
  closed shells   =     7
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./tce_n2_1re.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 n                          cc-pvdz                  6       15   3s2p1d



 Forming initial guess at       0.2s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -108.60004629

      Non-variational initial energy
      ------------------------------

 Total energy =    -109.123177
 1-e energy   =    -194.140722
 2-e energy   =      61.323154
 HOMO         =      -0.418440
 LUMO         =       0.037253


 Starting SCF solution at       0.2s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-10
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-10
 ----------------------------------------------


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -108.9433050654  4.28D-01  1.87D-01      0.2
                 2     -108.9544198128  1.04D-01  4.84D-02      0.2
                 3     -108.9551520697  1.60D-03  9.60D-04      0.4
                 4     -108.9551522080  5.12D-07  2.29D-07      0.6
                 5     -108.9551522080  2.00D-12  1.11D-12      0.9


       Final RHF  results 
       ------------------ 

         Total SCF energy =   -108.955152208013
      One-electron energy =   -194.370539428523
      Two-electron energy =     61.720996504843
 Nuclear repulsion energy =     23.694390715667

        Time for solution =      0.7s


             Final eigenvalues
             -----------------

              1      
    1  -15.6866
    2  -15.6831
    3   -1.4740
    4   -0.7735
    5   -0.6271
    6   -0.6099
    7   -0.6099
    8    0.1769
    9    0.1769
   10    0.5941
   11    0.8030
   12    0.8709
   13    0.8709
   14    0.8816
   15    1.0514
   16    1.0514
   17    1.1108

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    2  Occ=2.000000D+00  E=-1.568313D+01
              MO Center=  3.1D-19,  8.4D-19,  9.6D-13, r^2= 3.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.708251  1 N  s                 16     -0.708251  2 N  s          

 Vector    3  Occ=2.000000D+00  E=-1.474012D+00
              MO Center= -2.2D-16, -3.1D-16, -7.5D-16, r^2= 4.4D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.331729  1 N  s                 17      0.331729  2 N  s          
     6      0.218318  1 N  pz                21     -0.218318  2 N  pz         
     3      0.187745  1 N  s                 18      0.187745  2 N  s          

 Vector    4  Occ=2.000000D+00  E=-7.735351D-01
              MO Center=  2.1D-19, -2.4D-17, -5.1D-14, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.435587  1 N  s                 18     -0.435587  2 N  s          
     2      0.326753  1 N  s                 17     -0.326753  2 N  s          
     6     -0.217650  1 N  pz                21     -0.217650  2 N  pz         

 Vector    5  Occ=2.000000D+00  E=-6.271188D-01
              MO Center= -5.9D-16, -6.4D-16,  7.0D-14, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.453506  1 N  pz                21     -0.453506  2 N  pz         
     3     -0.342322  1 N  s                 18     -0.342322  2 N  s          
     9      0.211546  1 N  pz                24     -0.211546  2 N  pz         

 Vector    6  Occ=2.000000D+00  E=-6.099201D-01
              MO Center=  1.4D-15,  2.5D-15,  1.8D-15, r^2= 9.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.376337  1 N  py                20      0.376337  2 N  py         
     8      0.225781  1 N  py                23      0.225781  2 N  py         
     4      0.201370  1 N  px                19      0.201370  2 N  px         

 Vector    7  Occ=2.000000D+00  E=-6.099201D-01
              MO Center=  4.0D-16, -2.1D-16,  1.3D-15, r^2= 9.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.376337  1 N  px                19      0.376337  2 N  px         
     7      0.225781  1 N  px                22      0.225781  2 N  px         
     5     -0.201370  1 N  py                20     -0.201370  2 N  py         

 Vector    8  Occ=0.000000D+00  E= 1.769483D-01
              MO Center= -1.6D-16, -8.0D-17, -2.7D-15, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.680126  1 N  px                22     -0.680126  2 N  px         
     4      0.427345  1 N  px                19     -0.427345  2 N  px         

 Vector    9  Occ=0.000000D+00  E= 1.769483D-01
              MO Center= -5.8D-17, -2.3D-16, -1.8D-15, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.680126  1 N  py                23     -0.680126  2 N  py         
     5      0.427345  1 N  py                20     -0.427345  2 N  py         

 Vector   10  Occ=0.000000D+00  E= 5.940605D-01
              MO Center= -4.2D-17,  3.9D-16, -2.3D-13, r^2= 3.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      3.842185  1 N  s                 18     -3.842185  2 N  s          
     9      2.603045  1 N  pz                24      2.603045  2 N  pz         
     2      0.177647  1 N  s                 17     -0.177647  2 N  s          

 Vector   11  Occ=0.000000D+00  E= 8.030294D-01
              MO Center= -1.9D-15,  1.3D-15,  1.0D-13, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.844780  1 N  pz                24     -0.844780  2 N  pz         
     2     -0.542400  1 N  s                 17     -0.542400  2 N  s          
     3      0.456897  1 N  s                 18      0.456897  2 N  s          
     6     -0.332542  1 N  pz                21      0.332542  2 N  pz         
     1     -0.282062  1 N  s                 16     -0.282062  2 N  s          

 Vector   12  Occ=0.000000D+00  E= 8.708526D-01
              MO Center=  8.0D-15, -1.1D-15,  9.1D-14, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.634753  1 N  px                19      0.634753  2 N  px         
     7     -0.613669  1 N  px                22     -0.613669  2 N  px         
    12      0.177637  1 N  dxz               27     -0.177637  2 N  dxz        

 Vector   13  Occ=0.000000D+00  E= 8.708526D-01
              MO Center=  1.8D-15,  1.4D-14,  1.4D-13, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.634753  1 N  py                20      0.634753  2 N  py         
     8     -0.613669  1 N  py                23     -0.613669  2 N  py         
    14      0.177637  1 N  dyz               29     -0.177637  2 N  dyz        

 Vector   14  Occ=0.000000D+00  E= 8.815656D-01
              MO Center= -8.8D-15, -1.5D-14,  6.3D-14, r^2= 2.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.067905  1 N  s                 18      1.067905  2 N  s          
     2     -0.697744  1 N  s                 17     -0.697744  2 N  s          
     9     -0.470610  1 N  pz                24      0.470610  2 N  pz         
     6      0.416065  1 N  pz                21     -0.416065  2 N  pz         
     1     -0.317978  1 N  s                 16     -0.317978  2 N  s          

 Vector   15  Occ=0.000000D+00  E= 1.051362D+00
              MO Center= -1.6D-15, -5.5D-18, -9.0D-14, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.159518  1 N  px                22     -1.159518  2 N  px         
     4     -0.732883  1 N  px                19      0.732883  2 N  px         

 Vector   16  Occ=0.000000D+00  E= 1.051362D+00
              MO Center= -9.1D-18,  1.7D-15, -1.4D-13, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.159518  1 N  py                23     -1.159518  2 N  py         
     5     -0.732883  1 N  py                20      0.732883  2 N  py         

 Vector   17  Occ=0.000000D+00  E= 1.110833D+00
              MO Center=  1.3D-15, -1.4D-15,  1.4D-14, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.860459  1 N  s                 18     -1.860459  2 N  s          
     6      0.672798  1 N  pz                21      0.672798  2 N  pz         
    15     -0.375322  1 N  dzz               30      0.375322  2 N  dzz        
     2     -0.335897  1 N  s                 17      0.335897  2 N  s          
     9      0.299094  1 N  pz                24      0.299094  2 N  pz         


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
          29.942932617840           0.000000000000           0.000000000000
           0.000000000000          29.942932617840           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 N    7     7.00   2.00  0.85  0.90  2.13  1.05  0.06
    2 N    7     7.00   2.00  0.85  0.90  2.13  1.05  0.06

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0     -0.000000      0.000000     14.000000

     1   1 0 0     -0.000000      0.000000      0.000000
     1   0 1 0     -0.000000      0.000000      0.000000
     1   0 0 1     -0.000000      0.000000      0.000000

     2   2 0 0     -7.541334      0.000000      0.000000
     2   1 1 0     -0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -7.541334      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -8.663739      0.000000     14.968184

                   NWChem Extensible Many-Electron Theory Module
                   ---------------------------------------------

              ======================================================
                   This portion of the program was automatically
                  generated by a Tensor Contraction Engine (TCE).
                  The development of this portion of the program
                 and TCE was supported by US Department of Energy,
                Office of Science, Office of Basic Energy Science.
                      TCE is a product of Battelle and PNNL.
              Please cite: S.Hirata, J.Phys.Chem.A 107, 9887 (2003).
              ======================================================

            General Information
            -------------------
      Number of processors :     1
         Wavefunction type : Restricted Hartree-Fock
          No. of electrons :    14
           Alpha electrons :     7
            Beta electrons :     7
           No. of orbitals :    60
            Alpha orbitals :    30
             Beta orbitals :    30
        Alpha frozen cores :     0
         Beta frozen cores :     0
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity : singlet 
    Number of AO functions :    30
       Number of AO shells :    12
        Use of symmetry is : off
      Symmetry adaption is : off
         Schwarz screening : 0.10D-09

          Correlation Information
          -----------------------
          Calculation type : Coupled-cluster singles & doubles                           
   Perturbative correction : none                                                        
            Max iterations :      100
        Residual threshold : 0.10D-06
     T(0) DIIS level shift : 0.00D+00
     L(0) DIIS level shift : 0.00D+00
     T(1) DIIS level shift : 0.00D+00
     L(1) DIIS level shift : 0.00D+00
     T(R) DIIS level shift : 0.00D+00
     T(I) DIIS level shift : 0.00D+00
   CC-T/L Amplitude update :  5-th order DIIS
                I/O scheme : Global Array Library
        L-threshold :  0.10D-06
        EOM-threshold :  0.10D-06
 no EOMCCSD initial starts read in
 TCE RESTART OPTIONS
 READ_INT:   F
 WRITE_INT:  F
 READ_TA:    F
 WRITE_TA:   F
 READ_XA:    F
 WRITE_XA:   F
 READ_IN3:   F
 WRITE_IN3:  F
 SLICE:      F
 D4D5:       F
 quantumc:   F
 qrotation:  F
 pwtce:      F
 qducc:      F
 nacto:                         0
 nrotv:                         0
 nrot:                          0

            Memory Information
            ------------------
          Available GA space size is      26213500 doubles
          Available MA space size is      26212676 doubles

 Maximum block size        32 doubles

 tile_dim =     23

 Block   Spin    Irrep     Size     Offset   Alpha
 -------------------------------------------------
   1    alpha     a      7 doubles       0       1
   2    beta      a      7 doubles       7       1
   3    alpha     a     23 doubles      14       3
   4    beta      a     23 doubles      37       3

 Global array virtual files algorithm will be used

 Parallel file system coherency ......... OK

 Fock matrix recomputed
 1-e file size   =              900
 1-e file name   = ./tce_n2_1re.f1
 Cpu & wall time / sec            0.0            0.0

 tce_ao2e: fast2e=1
 half-transformed integrals in memory

 2-e (intermediate) file size =         2140200
 2-e (intermediate) file name = ./tce_n2_1re.v2i
 Cpu & wall time / sec            0.2            0.2

 tce_mo2e: fast2e=1
 2-e integrals stored in memory

 2-e file size   =          1356121
 2-e file name   = ./tce_n2_1re.v2
 Cpu & wall time / sec            0.0            0.0
 T1-number-of-tasks                    1

 t1 file size   =              161
 t1 file name   = ./tce_n2_1re.t1
 t1 file handle =       -999
 T2-number-of-boxes                    2

 t2 file size   =            51842
 t2 file name   = ./tce_n2_1re.t2
 t2 file handle =       -996
CCSD iterations
 ---------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall 
 ---------------------------------------------------------
NEW TASK SCHEDULING
CCSD_T1_NTS --- OK
CCSD_T2_NTS --- OK
    1   0.1578155467139  -0.3161625284392     0.0     0.0
    2   0.0430354207473  -0.3081074413652     0.0     0.0
    3   0.0157750005753  -0.3171279039955     0.0     0.0
    4   0.0077600178104  -0.3172768360512     0.0     0.0
    5   0.0038923278540  -0.3180983796177     0.0     0.0
 MICROCYCLE DIIS UPDATE:                    5                    5
    6   0.0007191916756  -0.3184632375899     0.0     0.0
    7   0.0002442977468  -0.3185098397295     0.0     0.0
    8   0.0001163637015  -0.3185097875883     0.0     0.0
    9   0.0000597661188  -0.3185111727472     0.0     0.0
   10   0.0000318661058  -0.3185123641020     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   10                    5
   11   0.0000036166183  -0.3185144833283     0.0     0.0
   12   0.0000012057093  -0.3185142743984     0.0     0.0
   13   0.0000004867685  -0.3185144259704     0.0     0.0
   14   0.0000002303917  -0.3185144311725     0.0     0.0
   15   0.0000001088090  -0.3185144481565     0.0     0.0
 MICROCYCLE DIIS UPDATE:                   15                    5
   16   0.0000000203082  -0.3185144573616     0.0     0.0
 -----------------------------------------------------------------
 Iterations converged
 CCSD correlation energy / hartree =        -0.318514457361618
 CCSD total energy / hartree       =      -109.273666665374549

 Singles contributions

 Doubles contributions

 Task  times  cpu:        1.4s     wall:        1.5s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls: 1105     1105     1.54e+04 2847     3725        0        0     4751     
number of processes/call 2.46e+14 -1.13e+15 -5.47e+14 0.00e+00 0.00e+00
bytes total:             8.17e+08 5.69e+07 2.22e+08 0.00e+00 0.00e+00 3.80e+04
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 27984976 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        16	        36
	current total bytes		         0	         0
	maximum total bytes		     80104	  22509768
	maximum total K-bytes		        81	     22510
	maximum total M-bytes		         1	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

          E. Aprà, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, Y. Alexeev, J. Anchell,
       V. Anisimov, F. W. Aquino, R. Atta-Fynn, J. Autschbach, N. P. Bauman,
     J. C. Becca, D. E. Bernholdt, K. Bhaskaran-Nair, S. Bogatko, P. Borowski,
        J. Boschen, J. Brabec, A. Bruner, E. Cauët, Y. Chen, G. N. Chuev,
      C. J. Cramer, J. Daily, M. J. O. Deegan, T. H. Dunning Jr., M. Dupuis,
   K. G. Dyall, G. I. Fann, S. A. Fischer, A. Fonari, H. Früchtl, L. Gagliardi,
      J. Garza, N. Gawande, S. Ghosh, K. Glaesemann, A. W. Götz, J. Hammond,
       V. Helms, E. D. Hermes, K. Hirao, S. Hirata, M. Jacquelin, L. Jensen,
   B. G. Johnson, H. Jónsson, R. A. Kendall, M. Klemm, R. Kobayashi, V. Konkov,
      S. Krishnamoorthy, M. Krishnan, Z. Lin, R. D. Lins, R. J. Littlefield,
      A. J. Logsdail, K. Lopata, W. Ma, A. V. Marenich, J. Martin del Campo,
   D. Mejia-Rodriguez, J. E. Moore, J. M. Mullin, T. Nakajima, D. R. Nascimento,
    J. A. Nichols, P. J. Nichols, J. Nieplocha, A. Otero-de-la-Roza, B. Palmer,
    A. Panyala, T. Pirojsirikul, B. Peng, R. Peverati, J. Pittner, L. Pollack,
   R. M. Richard, P. Sadayappan, G. C. Schatz, W. A. Shelton, D. W. Silverstein,
   D. M. A. Smith, T. A. Soares, D. Song, M. Swart, H. L. Taylor, G. S. Thomas,
            V. Tipparaju, D. G. Truhlar, K. Tsemekhman, T. Van Voorhis,
     Á. Vázquez-Mayagoitia, P. Verma, O. Villa, A. Vishnu, K. D. Vogiatzis,
        D. Wang, J. H. Weare, M. J. Williamson, T. L. Windus, K. Woliński,
        A. T. Wong, Q. Wu, C. Yang, Q. Yu, M. Zacharias, Z. Zhang, Y. Zhao,
                                and R. J. Harrison
                        "NWChem: Past, present, and future
                         J. Chem. Phys. 152, 184102 (2020)
                               doi:10.1063/5.0004997

                                      AUTHORS
                                      -------
     E. Apra, E. J. Bylaska, N. Govind, K. Kowalski, M. Valiev, W. A. de Jong,
      T. P. Straatsma, H. J. J. van Dam, D. Wang, T. L. Windus, N. P. Bauman,
       A. Panyala, J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec,
    K. Lopata, S. A. Fischer, S. Krishnamoorthy, M. Jacquelin, W. Ma, M. Klemm,
       O. Villa, Y. Chen, V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler,
           Eric Hermes, L. Jensen, J. E. Moore, J. C. Becca, V. Konjkov,
            D. Mejia-Rodriguez, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
        P.-D. Fan, A. Fonari, M. J. Williamson, R. J. Harrison, J. R. Rehr,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
      M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, M. Swart, Q. Wu,
   T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros,
     G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols,
       K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski,
       T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood,
      E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju,
        R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        1.4s     wall:        1.6s
